src/acmApi/authHelper.ts

```diff
@@ -0,0 +1,31 @@
+import axios, {AxiosInstance, AxiosRequestConfig} from "axios";
+
+function getBaseApi() {
+    return axios.create({
+        baseURL: process.env.REACT_APP_ACMAPI_URL,
```

Comment:
Handy to swap between `localhost` & `cloudfunctions.net`

Suggestion for automatically hot-swapping between them. Checking `process.env.NODE_ENV` and making a decision based on that. If the value is `production` then the URL would be the `...cloudfunctions.net` and if it is development then the URL would be `localhost.../`

```suggestion
        baseURL: (process.env.NODE_ENV !== "production" ? process.env.REACT_APP_LOCAL_URL : process.env.REACT_APP_PRODUCTION_URL)
```

Reply:
Probably good idea to extract the suggestion into a separate line just to make the code a bit more readable